client/signup/steps/site-type/index.jsx

```diff
@@ -109,6 +118,17 @@ export default connect(
 	( dispatch, ownProps ) => ( {
 		submitStep: ( siteTypeValue, themeRepo ) => {
 			dispatch( setSiteType( siteTypeValue ) );
+
+			let nextFlowName = ownProps.flowName;
+			if ( siteTypeValue === get( allSiteTypes, 'store.value', '' ) ) {
```

Comment:
We can probably leave these hardcoded to their siteType IDs for the time being (i.e. 'online-store' === siteType).